pkgs/top-level/python-packages.nix

```diff
@@ -8131,6 +8176,30 @@ in modules // {
     };
   };
 
+  django_raster = buildPythonPackage rec {
+    name = "django-raster-${version}";
+    version = "0.1.7a1";
```

Comment:
is there a specific reason you are not using a release?
https://pypi.python.org/pypi/django-raster/0.1.6